core/src/main/java/io/ultrabrew/metrics/data/ConcurrentIntTable.java

```diff
@@ -78,8 +76,8 @@ public ConcurrentIntTable(final int recordSize, int initialCapacity, final int m
 
   private ConcurrentIntTable(final int numAggFields, final int dataSize, int initialCapacity, final int maxCapacity, final long[] identity){
 
-    checkArgument(initialCapacity >= 0, "Illegal initial capacity");
-    checkArgument(maxCapacity >= initialCapacity, "max capacity should be greater than the initial capacity");
+    assert initialCapacity >= 0 : "Illegal initial capacity";
```

Comment:
how about renaming this class to `ConcurrentMonoidIntTable` and the other one to `ConcurrentMonoidLongTable`?